pyccel/parser/semantic.py

```diff
@@ -1446,18 +1449,25 @@ def _visit_CodeBlock(self, expr, **settings):
         return CodeBlock(ls)
 
     def _visit_Nil(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_EmptyNode(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_Break(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_Continue(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_Comment(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_CommentBlock(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
     def _visit_AnnotatedComment(self, expr, **settings):
+        expr.clear_user_nodes()
         return expr
```

Comment:
Could you please separate these methods from each other with an empty line?

Reply:
Fixed. See c62542a

Reply:
Thanks